src/graph/executor/algo/ProduceAllPathsExecutor.cpp

```diff
@@ -40,7 +40,7 @@ folly::Future<Status> ProduceAllPathsExecutor::execute() {
   futures.emplace_back(std::move(leftFuture));
   futures.emplace_back(std::move(rightFuture));
 
-  return folly::collect(futures)
+  return folly::collectAll(futures)
       .via(runner())
       .thenValue([this](auto&& status) {
```

Comment:
Are the stats here not of type std::vector<folly::Try<>>? Can you simply ignore the handling of Try?

Reply:
The type of the parameter is std::vector<folly::Try<Status>>
The buildPath function always returns Status::OK() unless the memory tracker runs out of exception（ will be caught by then Error）

Reply:
fixed